proposals/safe-fixed-sized-buffers.md

```diff
@@ -0,0 +1,115 @@
+Safe Fixed Size Buffers
+=====
+
+## Summary
+
+Provide a general-purpose and safe mechanism for declaring fixed sized buffers within C# classes, structs and interfaces.
```

Comment:
For interfaces, it's specifically `static fields` only, is that correct?

Reply:
I can assume that's implied since instance fields in interfaces aren't allowed (as far as the language is concerned, at least), but an explicit call-out would be good.

Reply:
> For interfaces, it's specifically `static fields` only, is that correct?

Yes, instance fields cannot be declared in an interface. I'll add an explicit note since they are not called fields in the speclet.